client/verta/verta/_deployment/endpoint.py

```diff
@@ -15,8 +16,11 @@ def __init__(self, conn, conf, workspace, id):
         self.id = id
 
     def __repr__(self):
-        # TODO: print full info
-        return "<Endpoint \"{}\">".format(self.path)
+        return '\n'.join((
+            "path: {}".format(self.path),
+            "id: {}".format(self.id),
+            "stage: {}".format(json.dumps(self.get_status(), indent=4)),
```

Comment:
I know I left this as an exercise for you 😆
The stage is kinda something we're trying to abstract away from the user for now, and also printing out the full status here (in JSON, too) may be a little overwhelming.

I'd bring `get_status()['status']`, `'date_created'`, `'date_updated'` up to the top level, and only show `'components'` here.

Reply:
Done-zo!